### DISCUSSION TITLE
Questions about the German translation

### DISCUSSION THREAD

**Discussion by Atque** [Stellarium/stellarium#3396] [Translations]
I'm not a fluent German speaker and certainly not active in any German-speaking astronomy community, but wouldn't the proper German name for Centaurus be _Zentaur_? And wouldn't Arcturus be called _Arktur_ and Vega _Wega_?

**Comment by alex-w** [comment]
Correct spelling for Vega and Arcturus.

**Comment by 10110111** [comment]
Wega, Zentaur, Arktur are all correct German names, which are the titles of corresponding Wikipedia.DE pages. But for Arktur and Wega these pages say, respectively, ["Arktur oder Arcturus"](https://de.wikipedia.org/wiki/Arktur) and ["Wega, auch Vega"](https://de.wikipedia.org/wiki/Wega). For the case of Centaurus the Latin name is explicitly specified as Latin: ["Zentaur oder Kentaur (lateinisch Centaurus)"](https://de.wikipedia.org/wiki/Zentaur_(Sternbild)).

**Comment by Atque** [comment]
> For the case of Centaurus the Latin name is explicitly specified as Latin: ["Zentaur oder Kentaur (lateinisch Centaurus)"](https://de.wikipedia.org/wiki/Zentaur_(Sternbild)).

Hmm, further down in the article, it says

> Die sternreiche Region des südlichen **Centaurus** enthält eine Reihe interessanter Beobachtungsobjekte.

Is there no consensus about what to call the constellation?

**Reply by 10110111** [reply]
I guess @gzotti should know the best about the frequency of use of different names.

**Reply by gzotti** [reply]
Matter of personal preference, seriously. German-speaking education systems have a long tradition of teaching Latin and even Greek, this is why Latin spelling is widespread and seen "more scientifical/original" by many. And with the new "official" star names, movement will almost certainly go into adopting those with IAU spelling.
